CHANGELOG.md

```diff
@@ -12,8 +13,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-* Add "traces" as legal event type. [#656](https://github.com/elastic/package-registry/pull/656)
-* Add input-level `template_path` field. [#655](https://github.com/elastic/package-registry/pull/655)
+- Add "traces" as legal event type. [#656](https://github.com/elastic/package-registry/pull/656)
```

Comment:
Out of curiosity - why did you refactored the CHANGELOG :) ? I'm not complaining, but maybe you can push it in a separate PR? /cc @ruflin

Reply:
Oh sorry I did not check, I think my editor format on save did this, I will revert this